CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # App Center SDK for Unity Editor Extensions Change Log
 
+## Development version
+
+* **[Bug fix]** Fixed excessive amount of GitHub API calls from unity extension.
```

Comment:
This is "unity extension" repo, may be just "the extension"?

Reply:
```suggestion
* **[Bug fix]** Fixed excessive amount of GitHub API calls.
```